packages/@aws-cdk/aws-appmesh/test/test.virtual-node.ts

```diff
@@ -251,7 +250,139 @@ export = {
+        // GIVEN
+        const stack = new cdk.Stack();
+
+        // WHEN
+        const mesh = new appmesh.Mesh(stack, 'mesh', {
+          meshName: 'test-mesh',
+        });
+
 
+        new appmesh.VirtualNode(stack, 'test-node', {
+          mesh,
+          dnsHostName: 'test',
+          backendDefaults: appmesh.ClientPolicy.acmTrust({
+            certificateAuthorityArns: ['path-to-certificate'],
+            enforceTls: true,
+            ports: [8080, 8081],
+          }),
+        });
+
+        // THEN
+        expect(stack).to(haveResourceLike('AWS::AppMesh::VirtualNode', {
+          Spec: {
+            BackendDefaults: {
+              ClientPolicy: {
+                TLS: {
+                  Enforce: true,
+                  Ports: [8080, 8081],
+                  Validation: {
+                    Trust: {
+                      ACM: {
+                        CertificateAuthorityArns: ['path-to-certificate'],
+                      },
+                    },
+                  },
+                },
+              },
+            },
+          },
+        }));
+        test.done();
+      },
+    },
+
+    'when a backend is added': {
+      'should add a backend visrtual service to the resource'(test: Test) {
+        // GIVEN
+        const stack = new cdk.Stack();
+
+        // WHEN
+        const mesh = new appmesh.Mesh(stack, 'mesh', {
+          meshName: 'test-mesh',
+        });
+
+        const node = new appmesh.VirtualNode(stack, 'test-node', {
+          mesh,
+          dnsHostName: 'test',
+        });
+
+        node.addListener(appmesh.VirtualNodeListener.tcp({
+          port: 80,
+          healthCheck: { timeout: cdk.Duration.seconds(3) },
+          timeout: { idle: cdk.Duration.seconds(10) },
+        }));
+
+        const service1 = new appmesh.VirtualService(stack, 'service-1', {
+          virtualServiceName: 'service1.domain.local',
+          mesh,
+          clientPolicy: appmesh.ClientPolicy.fileTrust({
+            certificateChain: ['path-to-certificate'],
+            enforceTls: true,
+            ports: [8080, 8081],
+          }),
+        });
+
+        node.addBackend(service1);
+
+        // THEN
+        expect(stack).to(haveResourceLike('AWS::AppMesh::VirtualNode', {
+          Spec: {
+            Listeners: [
+              {
+                HealthCheck: {
+                  HealthyThreshold: 2,
+                  IntervalMillis: 5000,
+                  Port: 80,
+                  Protocol: 'tcp',
+                  TimeoutMillis: 3000,
+                  UnhealthyThreshold: 2,
+                },
+                PortMapping: {
+                  Port: 80,
+                  Protocol: 'tcp',
+                },
+                Timeout: {
+                  TCP: {
+                    Idle: {
+                      Unit: 'ms',
+                      Value: 10000,
+                    },
+                  },
+                },
+              },
+            ],
```

Comment:
Is there a reason this is included in this test? I think we're only interested in the below `Backends` part, right?

Reply:
Removed Listeners and kept backend part in test.